suite-native/app/app.config.ts

```diff
@@ -49,7 +51,7 @@ const projectIds = {
 } as const satisfies Record<BuildType, string>;
 
 export default ({ config }: ConfigContext): ExpoConfig => {
-    const buildType = (process.env.EXPO_PUBLIC_ENVIRONMENT as BuildType) ?? 'debug';
+    const buildType = (getEnv() as BuildType) ?? 'debug';
```

Comment:
This will fail, you need to use that variable directly because in time of execution of this there no other env variables avaiable than ENV so Zod will fail on others.

Reply:
https://github.com/trezor/trezor-suite/pull/11720/commits/3dfc01061d1bf8a920aaacd9a2605ef516b56916